src/abstractdataframe/selection.jl

```diff
@@ -1064,139 +1050,137 @@ julia> gd = groupby(df, :a);
-│ 2   │ 2     │ 5     │ 7     │
-│ 3   │ 1     │ 2     │ 3     │
-│ 4   │ 1     │ 6     │ 7     │
-│ 5   │ 2     │ 3     │ 5     │
-│ 6   │ 2     │ 7     │ 9     │
-│ 7   │ 1     │ 4     │ 5     │
-│ 8   │ 1     │ 8     │ 9     │
+ Row │ b1     c1     b_c_+
+     │ Int64  Int64  Int64
+─────┼─────────────────────
+   1 │     2      1      3
+   2 │     2      5      7
+   3 │     1      2      3
+   4 │     1      6      7
+   5 │     2      3      5
+   6 │     2      7      9
+   7 │     1      4      5
+   8 │     1      8      9
 
 # broadcasting and column expansion
 julia> combine(gd, :b, AsTable([:b, :c]) => ByRow(extrema) => [:min, :max])
 8×4 DataFrame
-│ Row │ a     │ b     │ min   │ max   │
-│     │ Int64 │ Int64 │ Int64 │ Int64 │
-├─────┼───────┼───────┼───────┼───────┤
-│ 1   │ 1     │ 2     │ 1     │ 2     │
-│ 2   │ 1     │ 2     │ 2     │ 5     │
-│ 3   │ 2     │ 1     │ 1     │ 2     │
-│ 4   │ 2     │ 1     │ 1     │ 6     │
-│ 5   │ 3     │ 2     │ 2     │ 3     │
-│ 6   │ 3     │ 2     │ 2     │ 7     │
-│ 7   │ 4     │ 1     │ 1     │ 4     │
-│ 8   │ 4     │ 1     │ 1     │ 8     │
-
-# preventing vector from being spread across multiple rows
-julia> combine(gd, [:b, :c] .=> Ref)
+ Row │ a      b      min    max
+     │ Int64  Int64  Int64  Int64
+─────┼────────────────────────────
+   1 │     1      2      1      2
+   2 │     1      2      2      5
+   3 │     2      1      1      2
+   4 │     2      1      1      6
+   5 │     3      2      2      3
+   6 │     3      2      2      7
+   7 │     4      1      1      4
+   8 │     4      1      1      8
+
+julia> combine(gd, [:b, :c] .=> Ref) # preventing vector from being spread across multiple rows
 4×3 DataFrame
-│ Row │ a     │ b_Ref    │ c_Ref    │
-│     │ Int64 │ SubArra… │ SubArra… │
-├─────┼───────┼──────────┼──────────┤
-│ 1   │ 1     │ [2, 2]   │ [1, 5]   │
-│ 2   │ 2     │ [1, 1]   │ [2, 6]   │
-│ 3   │ 3     │ [2, 2]   │ [3, 7]   │
-│ 4   │ 4     │ [1, 1]   │ [4, 8]   │
+ Row │ a      b_Ref      c_Ref
+     │ Int64  SubArray…  SubArray…
+─────┼─────────────────────────────
+   1 │     1  [2, 2]     [1, 5]
+   2 │     2  [1, 1]     [2, 6]
+   3 │     3  [2, 2]     [3, 7]
+   4 │     4  [1, 1]     [4, 8]
 
 julia> combine(gd, AsTable(:) => Ref) # protecting result
 4×2 DataFrame
-│ Row │ a     │ a_b_c_Ref                            │
-│     │ Int64 │ NamedTuple…                          │
-├─────┼───────┼──────────────────────────────────────┤
-│ 1   │ 1     │ (a = [1, 1], b = [2, 2], c = [1, 5]) │
-│ 2   │ 2     │ (a = [2, 2], b = [1, 1], c = [2, 6]) │
-│ 3   │ 3     │ (a = [3, 3], b = [2, 2], c = [3, 7]) │
-│ 4   │ 4     │ (a = [4, 4], b = [1, 1], c = [4, 8]) │
+ Row │ a      a_b_c_Ref
+     │ Int64  NamedTup…
+─────┼──────────────────────────────────────────
+   1 │     1  (a = [1, 1], b = [2, 2], c = [1,…
+   2 │     2  (a = [2, 2], b = [1, 1], c = [2,…
+   3 │     3  (a = [3, 3], b = [2, 2], c = [3,…
+   4 │     4  (a = [4, 4], b = [1, 1], c = [4,…
```

Comment:
Why isn't this printed in full?

Reply:
Ah - good catch. This is not due to terminal width but the 32 character limit for the width of the column. Before I change it (to produce something narrower) let us confirm that we like this limit.

I am OK to crop in this case.

Reply:
Ah, indeed. This is for this kind of case where we have plenty of space that I wasn't a big fan of the hard limit at 32-char. But that's beyond the scope of this PR.